### PR TITLE
Add Scratch 2.0 → 3.0

### PR DESCRIPTION
**Pull request by mxmou**
**Resolves**

None

**Changes**

Adds the Scratch 2.0 → 3.0 addon.
Dark mode will be a separate addon, but #328 needs to be fixed first. 3.0 scratchblocks should be a separate addon too.

**Reason for changes**

#126 and #240

**Tests**

Seems to work on latest version of Chrome.

### CODE REVIEW

**Comment by JeffaloBob** [comment]
this is very exciting!! 🎉🎉

File: addons/scratchr2/addon.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    {
+      "url": "scratchr2.css",
+      "matches": [
+        "https://scratch.mit.edu/discuss/*",
+        "https://scratch.mit.edu/mystuff/",
+        "https://scratch.mit.edu/users/*",
+        "https://scratch.mit.edu/classes/*",
+        "https://scratch.mit.edu/studios/*",
+        "https://scratch.mit.edu/projects/*/remixes/",
+        "https://scratch.mit.edu/proejcts/*/studios/",
```
Comment (WorldLanguages) [inline addons/scratchr2/addon.json:21]: You misspelled "projects" D:

**Comment by WorldLanguages** [comment]
Just wondering - do you know why this happens?

Normal Scratch:
image
Scratch with this theme:
image

This specifically:
image

**Comment by GrahamSH-LLK** [comment]
Looks like a pfp from the next slide/row.

**Comment by WorldLanguages** [comment]
Btw @GrahamSH-LLK you fixed the wrong "proejcts", there were two of them :P I'll fix the second one

**Comment by WorldLanguages** [comment]
@Hans5958, do you know why the schema validation is failing?

**Comment by mxmou** [comment]
> Just wondering - do you know why this happens?
> image

I don't see that, even if I change the zoom level.

**Comment by WorldLanguages** [comment]
@mxmou Try going to my profile? I think it might depend on the length of the usernames.

**Comment by mxmou** [comment]
@WorldLanguages Unfortunately two more people seem to have followed you after you posted that screenshot.

Also, the arrows shouldn't be invisible. That's going to be a simple fix.

**Comment by WorldLanguages** [comment]
@mxmou Are you on Chrome(ium) 85 with Full HD resolution?

**Comment by mxmou** [comment]
@WorldLanguages Chrome 85, DPI scale is 125%. As I said, I tried on different zoom levels.

**Comment by GrahamSH-LLK** [comment]
Try logging in/out. There seems to be a caching thing.

**Comment by WorldLanguages** [comment]
@mxmou a little issue I found: go to https://scratch.mit.edu/users/@404/, the blue background remains, unlike other 404s.

**Pull request merged**